server_test.go

```diff
@@ -79,7 +81,7 @@ func startClient(network, addr string) {
 	duration := time.Duration((rand.Float64()*2+1)*float64(time.Second)) / 8
 	start := time.Now()
 	for time.Since(start) < duration {
-		sz := rand.Int()%(1024*1024) + 1
```

Comment:
Don't change the unit test

Reply:
I told u upper, test is not passing on OS macOS, its splitting on multiple package

Reply:
This is very strange. keep the original  unit test ，i will try to fix it

Reply:
pushed